scripts/oda_data.py

```diff
@@ -102,25 +102,47 @@ def read_gni():
         donor_name=lambda d: country_converter.convert(d.donor_name, to="short_name")
     )
 
-
-def read_idrc_estimates():
+def read_idrc_estimates() -> pd.DataFrame:
     """Read our estimates from the GoogleSheet which summarises the analysis"""
     url = (
         "https://docs.google.com/spreadsheets/d/e/2PACX-1vSqAIxjSZ78fE93CP1K9K0t8rL"
         "M2wi0z_nc60ezrUeDEIOPz-vr01SmmS_5nNnq_uPE0dM26m0V3rQK/pub?gid=1887834724"
         "&single=true&output=csv"
     )
     return (
-        pd.read_csv(url)
-        .assign(
-            donor_name=lambda d: country_converter.convert(d.iso_code, to="short_name"),
-            year=2022,
-        )
-        .rename(columns={"estimated_idrc": "idrc"})
-        .drop("iso_code", axis=1)
+        df=pd.read_csv(url).assign(donor_name=lambda d: country_converter.convert(d.iso_code, to="short_name"),
+            year=2022).rename(columns={"estimated_idrc": "idrc"}).drop("iso_code", axis=1)
     )
```

Comment:
This code wouldn't run, so always important to check that you're committing works.